scalatest/src/main/scala/org/scalatest/tools/RunnerJFrame.scala

```diff
@@ -665,7 +665,7 @@ private[scalatest] class RunnerJFrame(
     scalaTestMenu.add(exitItem)
     exitItem.addActionListener(
       new ActionListener() {
-        def actionPerformed(ae: ActionEvent): Unit = {
+        def actionPerformed(ae: ActionEvent | Null): Unit = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.